src/LICM.cpp

```diff
@@ -84,6 +84,12 @@ class LiftLoopInvariants : public IRMutator {
                 return false;
             }
         }
+        if (const Call *call = e.as<Call>()) {
+            if (call->is_intrinsic(Call::strict_float)) {
```

Comment:
Are there other intrinsics that also fit the bill here? eg, likely(), size_of_halide_buffer_t()?

Reply:
Done. Some of them don't exist at this stage of lowering, but if LICM is ever used earlier it would be good to handle things like likely.